### ISSUE TITLE
Cant enable MIDI Input devices

### ISSUE THREAD

**Issue by Hedronmx** [SubhadeepJasu/Ensembles#32]
Hello! When I enable midi input, i cant select any device. I currently own an Arturia Keylab 61 Essential and a Launchpad X. I cant activate neither of them.

**Comment by SubhadeepJasu** [comment]
Are you double clicking the name?

**Comment by Hedronmx** [comment]
I see the devices, I managed to activate the input by selecting the device and pressing space bar. But the app did not played any noise when pressing the midi keys, it then crashed

**Comment by SubhadeepJasu** [comment]
I see. Can you run the app from terminal and tell me what gets printed?

**Comment by Hedronmx** [comment]
No errors, it just crashes. No sound either when pressing the keys

```
carlosflores@MODEL-NAME-e8412f9a:~$ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=com.github.subhadeepjasu.ensembles --file-forwarding com.github.subhadeepjasu.ensembles @@u %U @@
Gtk-Message: 04:00:47.707: Failed to load module "canberra-gtk-module"
Gtk-Message: 04:00:47.709: Failed to load module "canberra-gtk-module"
fluidsynth: Using PulseAudio driver
fluidsynth: Using PulseAudio driver
fluidsynth: Using PulseAudio driver
Ticks: ///////// 480 //////
Time Signature = 3/4
Style: /// Config:1;90,0 0
Style: /// IntroA:2 1440
Style: /// IntroB:4 4320
Style: /// VarA:12 15840
Style: /// FillA:14 18720
Style: /// VarB:15 20160
Style: /// FillB:17 23040
Style: /// VarC:18 24480
Style: /// FillC:20 27360
Style: /// VarD:21 28800
Style: /// FillD:23 31680
Style: /// EndingA:24 33120
Style: /// EOS1:28 38880
Style: /// EndingB:29 40320
Style: /// EOS2:34 47520
Ticks: ///////// 480 //////
Time Signature = 4/4
Style: /// Config:1;125,1 0
Style: /// IntroA:2 1920
Style: /// IntroB:4 5760
Style: /// VarA:12 21120
Style: /// FillA:14 24960
Style: /// VarB:15 26880
Style: /// FillB:17 30720
Style: /// VarC:18 32640
Style: /// FillC:20 36480
Style: /// VarD:21 38400
Style: /// FillD:23 42240
Style: /// EndingA:24 44160
Style: /// EOS:28 51840
Style: /// EndingB:32 59520
Style: /// EOS:40 74880
Ticks: ///////// 480 //////
Time Signature = 4/4
Style: /// Config:1;125,1 0
Style: /// IntroA:2 1920
Style: /// IntroB:4 5760
Style: /// VarA:12 21120
Style: /// FillA:14 24960
Style: /// VarB:15 26880
Style: /// FillB:17 30720
Style: /// VarC:18 32640
Style: /// FillC:20 36480
Style: /// VarD:21 38400
Style: /// FillD:23 42240
Style: /// EndingA:24 44160
Style: /// EOS:26 48000
Style: /// EndingB:32 59520
Style: /// EOS:40 74880
Ticks: ///////// 480 //////
Time Signature = 4/4
Style: /// Config:1;100,1 0
Style: /// IntroA:2 1920
Style: /// IntroB:4 5760
Style: /// VarA:12 21120
Style: /// FillA:14 24960
Style: /// VarB:15 26880
Style: /// FillB:17 30720
Style: /// VarC:18 32640
Style: /// FillC:20 36480
Style: /// VarD:21 38400
Style: /// FillD:23 42240
Style: /// EndingA:24 44160
Style: /// EOS:26 48000
Style: /// EndingB:32 59520
Style: /// EOS:41 76800
Ticks: ///////// 480 //////
Time Signature = 4/8
Style: /// Config:1;80,1 0
Style: /// Intro1:2 960
Style: /// Intro2:6 4800
Style: /// VarA:22 20160
Style: /// FillA:26 24000
Style: /// VarB:28 25920
Style: /// FillB:32 29760
Style: /// VarC:34 31680
Style: /// FillC:38 35520
Style: /// VarD:40 37440
Style: /// FillD:44 41280
Style: /// Ending1:46 43200
Style: /// EOS:50 47040
Style: /// Ending2:55 51840
Style: /// EOS:60 56640
Ticks: ///////// 480 //////
Time Signature = 4/4
Style: /// Config:1;110,1 0
Style: /// IntroA:2 1920
Style: /// IntroB:4 5760
Style: /// VarA:12 21120
Style: /// FillA:14 24960
Style: /// VarB:15 26880
Style: /// FillB:17 30720
Style: /// VarC:18 32640
Style: /// FillC:20 36480
Style: /// VarD:21 38400
Style: /// FillD:23 42240
Style: /// EndingA:24 44160
Style: /// EOS:28 51840
Style: /// EndingB:32 59520
Style: /// EOS:40 74880
ts: 4
fluidsynth: Using PulseAudio driver

** (com.github.subhadeepjasu.ensembles:2): WARNING **: 04:00:49.014: MediaKeyListener.vala:61: Could not grab media player keys: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
chan...to /app/share/com.github.subhadeepjasu.ensembles/StyleFiles/Ball_Room@French_Waltz.enstl
Ticks: ///////// 480 //////
Time Signature = 3/4
Style: /// Config:1;90,0 0
Style: /// IntroA:2 1440
Style: /// IntroB:4 4320
Style: /// VarA:12 15840
Style: /// FillA:14 18720
Style: /// VarB:15 20160
Style: /// FillB:17 23040
Style: /// VarC:18 24480
Style: /// FillC:20 27360
Style: /// VarD:21 28800
Style: /// FillD:23 31680
Style: /// EndingA:24 33120
Style: /// EOS1:28 38880
Style: /// EndingB:29 40320
Style: /// EOS2:34 47520
changing...to /app/share/com.github.subhadeepjasu.ensembles/StyleFiles/Ball_Room@French_Waltz.enstl
e:
f:
90 >>>>
g:
ts: 4
h
Connected
Velocity: 26, Key: 40, Type:146, Raw: 1b2892
Velocity: 37, Key: 40, Type:130, Raw: 262882
Velocity: 40, Key: 40, Type:146, Raw: 292892
Velocity: 54, Key: 40, Type:130, Raw: 372882
Velocity: 36, Key: 43, Type:146, Raw: 252b92
Velocity: 29, Key: 36, Type:146, Raw: 1e2492
Velocity: 30, Key: 43, Type:130, Raw: 1f2b82
Velocity: 35, Key: 38, Type:146, Raw: 242692
Velocity: 32, Key: 40, Type:146, Raw: 212892
Velocity: 74, Key: 36, Type:130, Raw: 4b2482
Velocity: 56, Key: 43, Type:146, Raw: 392b92
Velocity: 35, Key: 40, Type:130, Raw: 242882
Velocity: 39, Key: 38, Type:130, Raw: 282682
Velocity: 36, Key: 43, Type:130, Raw: 252b82
Velocity: 49, Key: 36, Type:146, Raw: 322492
Velocity: 54, Key: 43, Type:146, Raw: 372b92
Velocity: 68, Key: 36, Type:130, Raw: 452482
Velocity: 85, Key: 43, Type:130, Raw: 562b82
Velocity: 42, Key: 50, Type:146, Raw: 2b3292
Velocity: 41, Key: 50, Type:130, Raw: 2a3282
Velocity: 72, Key: 52, Type:146, Raw: 493492
Velocity: 34, Key: 47, Type:146, Raw: 232f92
Velocity: 61, Key: 52, Type:130, Raw: 3e3482
Velocity: 33, Key: 48, Type:146, Raw: 223092
Velocity: 9, Key: 47, Type:130, Raw: a2f82
Velocity: 79, Key: 52, Type:146, Raw: 503492
Velocity: 43, Key: 48, Type:130, Raw: 2c3082
Velocity: 52, Key: 52, Type:130, Raw: 353482
Velocity: 64, Key: 52, Type:146, Raw: 413492
Velocity: 50, Key: 52, Type:130, Raw: 333482
Velocity: 61, Key: 52, Type:146, Raw: 3e3492
Velocity: 7, Key: 45, Type:146, Raw: 82d92
Velocity: 27, Key: 45, Type:130, Raw: 1c2d82
Velocity: 9, Key: 52, Type:130, Raw: a3482
Velocity: 59, Key: 50, Type:146, Raw: 3c3292
Velocity: 46, Key: 48, Type:146, Raw: 2f3092
Velocity: 37, Key: 48, Type:130, Raw: 263082
Velocity: 67, Key: 50, Type:130, Raw: 443282
Velocity: 37, Key: 33, Type:146, Raw: 262192

```

**Comment by SubhadeepJasu** [comment]
Thank you very much. I gotta investigate the output a bit. It looks the keyboard is incompatible with ensembles in it's current state. I need to modify some things to make it compatible. 

On a sidenote, are you on discord or slack or twitter? We can take this conversation to DMs if you like. Because working closely with you to provide support for this keyboard.

**Comment by Hedronmx** [comment]
This is my discord geop0p3#6842

**Comment by SubhadeepJasu** [comment]
Thanks! Meet you there. :)

**Comment by SubhadeepJasu** [comment]
Closing, as its fixed in #29 . Opening new issue to track onboard knobs and faders compatibility.
